lib/pbench/agent/modules/metaclass.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
```

Comment:
`metaclass.py` seems much too generic a name for this file: shouldn't it be named `toolmetadata.py`?

Reply:
Yeah, my original thought was to avoid giving the module and class the same name, but it probably makes it more understandable to change it